static/app/components/events/eventStatisticalDetector/geoLocationDiff.spec.tsx

```diff
@@ -0,0 +1,159 @@
+import {Event as EventMock} from 'sentry-fixture/event';
+
+import {render, screen, userEvent, within} from 'sentry-test/reactTestingLibrary';
+
+import {Event} from 'sentry/types';
+import {DAY} from 'sentry/utils/formatters';
+
+import GeoLocationDiff from './geoLocationDiff';
+
+describe('GeoLocationDiff', () => {
+  const DURATION_REGRESSION_TYPE = 1017;
+  const PROJECT_ID = '1';
+
+  // Breakpoint is stored as seconds
+  const BREAKPOINT_TIMESTAMP = (Date.now() - 7 * DAY) / 1000;
+  let mockEvent: Event;
+
+  beforeEach(() => {
+    mockEvent = {
+      ...EventMock(),
+      occurrence: {
```

Comment:
nit: can pass overrides into the fixture like this: `const mockEvent = EventMock({occurrence: {...}})`